rust/processor/src/schema.rs

```diff
@@ -268,6 +270,25 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    current_fungible_asset_balances (storage_id) {
+        #[max_length = 66]
+        storage_id -> Varchar,
```

Comment:
This is meant to be of the format "0x1::aptos_coin::AptosCoin" for Coins and "0x123456" for FungibleAssets right? I'd prefer a more descriptive name such as asset_id, which is also more consistent with token_data_id or collection_id for token v2 tables

Reply:
It's not meant to be asset_id. For v1 this is a hash of owner address + asset type.